seacrowd/sea_datasets/khpos/khpos.py

```diff
@@ -0,0 +1,211 @@
+# limitations under the License.
+
+"""
+The khPOS Corpus (Khmer POS Corpus) is a 12,000 sentences (25,626 words) manually word segmented and POS tagged corpus 
+developed for Khmer language NLP research and developments. We collected Khmer sentences from websites that include
+various area such as economics, news, politics. Moreover it is also contained some student list and voter list of 
+national election committee of Cambodia. The average number of words per sentence in the whole corpus is 10.75. 
+Here, some symbols such as "។" (Khmer sign Khan), "៖" (Khmer sign Camnuc pii kuuh), "-", "?", "[", "]" etc. also 
+counted as words. The shortest sentence contained only 1 word and longest sentence contained 169 words. This dataset contains
+A validation set and a test set, each containing 1000 sentences.
+"""
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@inproceedings{kyaw2017comparison,
+  title={Comparison of Six POS Tagging Methods on 12K Sentences Khmer Language POS Tagged Corpus},
+  author={Ye Kyaw Thu and Vichet Chea and Yoshinori Sagisaka},
+  booktitle={Proceedings of the first Regional Conference on Optical character recognition and Natural language processing technologies for ASEAN languages (ONA 2017)},
+  year={2017},
+  month={December 7-8},
+  address={Phnom Penh, Cambodia}
+}
+"""
+
+_DATASETNAME = "khpos"
+
+_DESCRIPTION = """\
+The khPOS Corpus (Khmer POS Corpus) is a 12,000 sentences (25,626 words) manually word segmented and POS tagged corpus 
+developed for Khmer language NLP research and developments. We collected Khmer sentences from websites that include
+various area such as economics, news, politics. Moreover it is also contained some student list and voter list of 
+national election committee of Cambodia. The average number of words per sentence in the whole corpus is 10.75. 
+Here, some symbols such as "។" (Khmer sign Khan), "៖" (Khmer sign Camnuc pii kuuh), "-", "?", "[", "]" etc. also 
+counted as words. The shortest sentence contained only 1 word and longest sentence contained 169 words. This dataset contains
+A validation set and a test set, each containing 1000 sentences.
+"""
+
+_HOMEPAGE = "https://github.com/ye-kyaw-thu/khPOS/tree/master"
+
+_LANGUAGES = ['khm']  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value 
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://raw.githubusercontent.com/ye-kyaw-thu/khPOS/master/corpus-draft-ver-1.0/data/after-replace/train.all2",
+    'validation': "https://raw.githubusercontent.com/ye-kyaw-thu/khPOS/master/corpus-draft-ver-1.0/data/OPEN-TEST",
+    'test'      : "https://raw.githubusercontent.com/ye-kyaw-thu/khPOS/master/corpus-draft-ver-1.0/data/CLOSE-TEST"
+}
```

Comment:
I think it's better to structure it like this,
```
_URLS = {
    _DATASETNAME: {
        "train": ...,
        "validation": ...,
        "test": ...,
    },
}
```
wdyt?

Reply:
Yeah, this is definitely better! Changed!